src/QTKit/QTCompat.cs

```diff
@@ -0,0 +1,2737 @@
+// autogenerated stub - do not edit
+#if !XAMCORE_4_0
+
+#pragma warning disable 0067
+
+namespace QTKit {
+
+	[ObjCRuntime.Obsoleted (ObjCRuntime.PlatformName.MacOSX, 10,15, message: ObjCRuntime.Constants.MacOS32bitsUnavailable)]
+	public class QTCaptureConnection : Foundation.NSObject {
+		public Foundation.NSObject GetAttribute (System.String attributeKey)
+		{
+			return default (Foundation.NSObject);
+		}
+
+		public void SetAttribute (Foundation.NSObject attribute, System.String key)
+		{
+		}
+
+		public QTCaptureConnection ()
+		{
+			throw new System.NotImplementedException (ObjCRuntime.Constants.MacOS32bitsUnavailable);
+		}
+
+		protected QTCaptureConnection (Foundation.NSObjectFlag t)
+		{
+			throw new System.NotImplementedException (ObjCRuntime.Constants.MacOS32bitsUnavailable);
+		}
+
+		protected internal QTCaptureConnection (System.IntPtr handle)
+		{
+			throw new System.NotImplementedException (ObjCRuntime.Constants.MacOS32bitsUnavailable);
+		}
+
+		public virtual Foundation.NSObject GetAttribute (Foundation.NSString attributeKey)
+		{
+			return default (Foundation.NSObject);
+		}
+
+		public virtual System.Boolean IsAttributeReadOnly (System.String attributeKey)
+		{
+			return default (System.Boolean);
+		}
+
+		public virtual void SetAttribute (Foundation.NSObject attribute, Foundation.NSString key)
+		{
+		}
+
+		public QTKit.QTMediaType MediaTypeValue { get; }
+
+		public override System.IntPtr ClassHandle { get; }
+
+		public virtual Foundation.NSDictionary ConnectionAttributes { get; set; }
+
+		public virtual System.Boolean Enabled { get; set; }
+
+		public virtual QTKit.QTFormatDescription FormatDescription { get; }
+
+		public virtual System.String MediaType { get; }
+
+		public virtual Foundation.NSObject Owner { get; }
+
+		static public Foundation.NSString AttributeDidChangeNotification { get; }
```

Comment:
Not a big deal, but most of our codebase uses `public static` and not `static public`.

Reply:
not sure we are very consistent in our sources, but it's an easy change